fuel/downloaders/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Download modules for built-in datasets.
+
+Download methods accept a single argument, `save_directory`, which tells
+where to save the downloaded files.
+
+"""
+from fuel.downloaders.binarized_mnist import binarized_mnist
+
+__all__ = ('binarized_mnist',)
```

Comment:
Could you turn this into a list? Same for the `converters/__init__.py` actually. The Python specification says it should be a list, and I've never seen a tuple used actually, so it might lead to some strange behavior down the road (maybe?)